internal/pkg/aws/elbv2/elbv2_test.go

```diff
@@ -144,33 +144,37 @@ func TestELBV2_TargetsHealth(t *testing.T) {
 }
 
 func TestELBV2_ListenerRuleHostHeaders(t *testing.T) {
-	mockARN := "mockListenerRuleARN"
+	mockARN1 := "mockListenerRuleARN1"
+	mockARN2 := "mockListenerRuleARN2"
 	testCases := map[string]struct {
-		setUpMock func(m *mocks.Mockapi)
-
+		setUpMock   func(m *mocks.Mockapi)
+		inmockARNs  []string
```

Comment:
nit:
```suggestion
		inARNs  []string
```
I think the `mock` keyword should be added to dependencies that are assumed to work, not to the input of the functions. I'd think of a function as a black box that maps an input to the output; the dependencies are inside of the black box and need to be mocked 🤔 💭